scripts/update-reg-repo.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+# Write output to error output stream.
+echo_stderr() {
+  echo -e "${1}" >&2
+}
+
+die()
+{
+  local _return="${2:-1}"
+  echo_stderr "$1"
+  exit "${_return}"
+}
+
+set -euo pipefail
+
+SCRIPT_DIR="$(dirname "$(realpath "${BASH_SOURCE[0]:-"$0"}")")"
+ROOT_DIR="$SCRIPT_DIR/.."
+CHART_DIR="$ROOT_DIR/deploy/helm/charts"
+VALUES_YAML="$CHART_DIR/values.yaml"
+
+NEW_REGISTRY="ghcr.io"
+NEW_REPOSITORY="openebs/dev"
```

Comment:
this is the namespace not the repository?

Reply:
Yeah namespace, but in the values it's repository that's why kept it same so that there is no confusion.

Reply:
Yeah but here it's not the repository it's the namespace, the repository would be openebs/dev/zfs-driver

Reply:
It is namespace but the value that we set on the values.yaml is termed as repository and not namespace.

Reply:
What I meant is, it must be consistent. You've renamed it but it's still not consistent IMHO.
If the parameter name is repository, then we should pass the entire thing "openebs/dev/zfs-driver" rather than "openebs/dev" which is a namespace
If the parameter name is namespace, then we should only pass "openebs/dev", and let the script add the "zfs-driver", rather than passing "openebs/dev/zfs-driver" which is not a namespace

Reply:
I have removed the defaults and made the args compulsory,  so I will revert it back to repository and expect repository to be passed.

Reply:
Done.